style/docs.css

```diff
@@ -127,6 +130,7 @@ ul {
   margin-top: 1rem;
   margin-bottom: 2rem;
 }
+
```

Comment:
what is this "+" for?

Reply:
That is added by Github, just suggesting us to add an extra line at the end of each file